webextension/manifest.json

```diff
@@ -35,6 +35,14 @@
         "script": "forceOpenPageActionPopup.js",
         "paths": [["experiments", "pageAction"]]
       }
+    },
+    "study": {
```

Comment:
As a convention, we are doing a 'privileged/{theApi}' directory tree.  Not sure if it's better or not, to make it easy to know where the gross stuff is, and eventually know that known md5s for files are "known qa'd"

Reply:
Agreed, I was planning on a vNext update that moves them into a subdirectory named `experimentalAPIs`, but if `privileged` is already a convention then I don't mind using that.